### ISSUE TITLE
Doesnt start

### ISSUE THREAD

**Issue by daniel-lucio** [fwbuilder/fwbuilder#60]
This happens with FWB 5.0 and 5.3.7 as well. 

$ fwbuilder 
Firewall Builder GUI 5.3.7
<gets frozen>

Any suggestion?
Under Mageia 6 with QT 5.9.4

**Comment by alienbrainsucker** [comment]
Cloned from git and compiled from source ?
Did you run "sudo make install" after compiling the source ?
I had the same problem when i tried to start the binary directly from the src/gui directory.

When running "make install" you'll get a new directory /usr/local/share/fwbuilder which is needed by the binary. The fwbuilder binary is installed in /usr/local/bin/fwbuilder (on my fedora26 system).

Long story short: after "sudo make install" the fwbuilder gui should pop up.

**Comment by daniel-lucio** [comment]
I fixed, it was a lost symbol. Rebuilding the RPM fixed it.
